test/lib/ansible_test/_data/setup/docker.sh

```diff
@@ -6,9 +6,8 @@ set -eu
 rm -f /usr/sbin/policy-rc.d
 
 # Improve prompts on remote host for interactive use.
-# shellcheck disable=SC1117
-cat << EOF > ~/.bashrc
-alias ls='ls --color=auto'
-export PS1='\[\e]0;\u@\h: \w\a\]\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
-cd ~/ansible/
-EOF
+# `cat << EOF > ~/.bashrc` flakes sometimes since /tmp may not be ready yet in
+# the container. So don't do that
+echo "alias ls='ls --color=auto'" >> ~/.bashrc
```

Comment:
Should this first line be `>` instead of `>>` to match the previous behavior?

Reply:
Good catch. Yes, it should be.

Reply:
done

Reply:
done